<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>750</width>
                <height>433</height>
            </rect>
        </property>
        <property name="minimumSize">
            <size>
                <width>750</width>
                <height>433</height>
            </size>
        </property>
        <property name="maximumSize">
            <size>
                <width>750</width>
                <height>650</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>Лабораторная работа 14: Криптоанализ полиалфавитных шифров</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <widget class="QTextEdit" name="plain_text">
                <property name="geometry">
                    <rect>
                        <x>240</x>
                        <y>10</y>
                        <width>501</width>
                        <height>191</height>
                    </rect>
                </property>
                <property name="inputMethodHints">
                    <set>Qt::ImhMultiLine</set>
                </property>
            </widget>
            <widget class="QTextEdit" name="cipher_text">
                <property name="geometry">
                    <rect>
                        <x>240</x>
                        <y>210</y>
                        <width>501</width>
                        <height>191</height>
                    </rect>
                </property>
            </widget>
            <widget class="QTableWidget" name="table_stats">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>130</y>
                        <width>221</width>
                        <height>271</height>
                    </rect>
                </property>
                <property name="mouseTracking">
                    <bool>true</bool>
                </property>
                <property name="horizontalScrollBarPolicy">
                    <enum>Qt::ScrollBarAlwaysOff</enum>
                </property>
                <property name="sizeAdjustPolicy">
                    <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
                </property>
                <property name="editTriggers">
                    <set>QAbstractItemView::NoEditTriggers</set>
                </property>
                <property name="selectionMode">
                    <enum>QAbstractItemView::SingleSelection</enum>
                </property>
                <property name="selectionBehavior">
                    <enum>QAbstractItemView::SelectRows</enum>
                </property>
                <property name="sortingEnabled">
                    <bool>true</bool>
                </property>
                <property name="columnCount">
                    <number>3</number>
                </property>
                <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
                    <bool>false</bool>
                </attribute>
                <attribute name="verticalHeaderVisible">
                    <bool>false</bool>
                </attribute>
                <column>
                    <property name="text">
                        <string>Символ</string>
                    </property>
                </column>
                <column>
                    <property name="text">
                        <string>Количество</string>
                    </property>
                </column>
                <column>
                    <property name="text">
                        <string>%</string>
                    </property>
                </column>
            </widget>
            <widget class="QFrame" name="horizontalFrame">
                <property name="geometry">
                    <rect>
                        <x>-30</x>
                        <y>360</y>
                        <width>811</width>
                        <height>291</height>
                    </rect>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout"/>
            </widget>
            <widget class="QComboBox" name="combo_method">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>10</y>
                        <width>221</width>
                        <height>21</height>
                    </rect>
                </property>
                <property name="currentIndex">
                    <number>0</number>
                </property>
                <item>
                    <property name="text">
                        <string>Метод индекса совпадений</string>
                    </property>
                </item>
                <item>
                    <property name="text">
                        <string>Автокорреляционный метод</string>
                    </property>
                </item>
                <item>
                    <property name="text">
                        <string>Метод Касиски</string>
                    </property>
                </item>
            </widget>
            <widget class="QLineEdit" name="line_abc">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>40</y>
                        <width>141</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="placeholderText">
                    <string>Алфавит</string>
                </property>
            </widget>
            <widget class="QLineEdit" name="line_key">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>70</y>
                        <width>141</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="placeholderText">
                    <string>Ключ</string>
                </property>
            </widget>
            <widget class="QSpinBox" name="spin_key_len">
                <property name="geometry">
                    <rect>
                        <x>160</x>
                        <y>70</y>
                        <width>71</width>
                        <height>20</height>
                    </rect>
                </property>
            </widget>
            <widget class="QPushButton" name="btn_analyse">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>100</y>
                        <width>111</width>
                        <height>23</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Анализировать</string>
                </property>
            </widget>
            <widget class="QPushButton" name="btn_decrypt">
                <property name="geometry">
                    <rect>
                        <x>120</x>
                        <y>100</y>
                        <width>111</width>
                        <height>23</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Расшифровать</string>
                </property>
            </widget>
            <widget class="QComboBox" name="combo_language">
                <property name="geometry">
                    <rect>
                        <x>160</x>
                        <y>40</y>
                        <width>71</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="currentIndex">
                    <number>0</number>
                </property>
                <item>
                    <property name="text">
                        <string>Русский</string>
                    </property>
                </item>
                <item>
                    <property name="text">
                        <string>Английский</string>
                    </property>
                </item>
            </widget>
            <zorder>horizontalFrame</zorder>
            <zorder>plain_text</zorder>
            <zorder>cipher_text</zorder>
            <zorder>table_stats</zorder>
            <zorder>combo_method</zorder>
            <zorder>line_abc</zorder>
            <zorder>line_key</zorder>
            <zorder>spin_key_len</zorder>
            <zorder>btn_analyse</zorder>
            <zorder>btn_decrypt</zorder>
            <zorder>combo_language</zorder>
        </widget>
        <widget class="QMenuBar" name="menuBar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>750</width>
                    <height>21</height>
                </rect>
            </property>
            <widget class="QMenu" name="menu_text">
                <property name="title">
                    <string>Текст</string>
                </property>
                <addaction name="action_open"/>
                <addaction name="action_save"/>
            </widget>
            <addaction name="menu_text"/>
        </widget>
        <action name="action_open">
            <property name="text">
                <string>Открыть</string>
            </property>
        </action>
        <action name="action_save">
            <property name="text">
                <string>Сохранить</string>
            </property>
        </action>
    </widget>
    <resources/>
    <connections/>
</ui>
